pkgs/development/libraries/linbox/default.nix

```
@@ -74,7 +74,7 @@ stdenv.mkDerivation rec {
     inherit version;
     description = "C++ library for exact, high-performance linear algebra";
     license = stdenv.lib.licenses.lgpl21Plus;
-    maintainers = [stdenv.lib.maintainers.timokau];
+    maintainers = with stdenv.lib.maintainers; teams.sage.members;
```

```suggestion
    maintainers = with stdenv.lib.maintainers; [ ] ++ teams.sage.members;
```
or just ``stdenv.lib.teams.sage.members``

---

As mentioned in the other thread, I was confused myself and thought it was `lib.maintainers.teams`. Then the `with maintainers` would have made it easier to add further maintainers. Changed now.